frontend/pages/_id/index.vue

```diff
@@ -23,7 +23,7 @@
     <div class="container">
       <div :class="['grid', annotationEnabled ? 'grid--editable' : '']">
         <Results :dataset="dataset" :headerHeight="headerHeight"> </Results>
-        <SideBar :dataset="dataset" />
+        <SideBar :dataset="dataset" :entities="dataset.entities" />
```

Comment:
we shouldn't include a task specific property here. For text classification there is no `entities` property defined and this should bring unexpected errors